literature/nakamura_2014_structural_22136/nakamura_2014_structural_22136_p1_1_inset.yaml

```diff
@@ -0,0 +1,102 @@
+    comment: Focus on counterion (Cs+) to electrode distance as a function of potential # Add whatever you think is important to understand the data
+
+electrochemical system:
+    version: 1
+    instrumentation:
+        - type:
+          supplier: 
+        - type:
+          model:
+          supplier:
+        - type:
+          model:
+          supplier:
+    electrolyte:
+        type: # aqueous = aq, non-aqueous=naq
+        components: # create as many components as necessary. A simple electrolyte consists of two components, i.e., H2SO4 and water
+            - name: CsBr # can be trivia name, sum formula, etc
+              concentration:
+                  value: 0.1
+                  unit: M #[M, mM, µM, g kg-1, ...]
+              type: #
+              source:
+                  supplier:
+                  LOT: 
+            - name: water
+              type: solvent
+              proportion:
+                  value:
+                  unit:
+              source:
+                  refinement:
+                  quality:
+                  total organic carbon: 
+                      value:
+                      unit:
+            - name: 
+              type:
+              proportion:
+                  value: 
+                  unit:
+            - name: CsOH
+              type: base
+              source:
+                  supplier:
+                  LOT: 
+              concentration:
+                  value: 0.05
+                  unit: M #[M, mM, µM, g kg-1, ...] 
+        ph: 
+            value: 12.5
+            uncertainty:
+        temperature: # provide the temperature as it is given in the article. Assume room temperature if nothing is provided
+            value: 298.15 
+            unit: K # [K, C, F]
+    electrodes:
+        configuration: 3 # number of electrodes
+        reference electrode:
+            type: Ag/AgCl # the reference electrode given in the experimental section. Might differ from that in the plot, MSE-sat, MSE-3M, MSE-1M, RHE, AgCl2
+            source:
+                supplier:
+        counter electrode:
+            material:
+            crystallographic orientation:
+            shape:
+        working electrode:
+            material: Ag
+            crystallographic orientation: '100' # hkl, 100, 110, poly
+            source:
+                supplier:
+                LOT: 
+            shape: # optional
+                type: sphere # bead/sphere
+                height:
+                    value:
+                    unit:
+                diameter:
+                    value: 10
+                    unit: mm
+            geometric electrolyte contact area: 
+                value:
+                unit:
+            preparation procedure: Ag electrode etched with chromic acid, then annealed in H2/Ar at 700K with inductively heated furnace. # Short description of sample preparation. This can be very useful. :)

```

Comment:
```suggestion
            preparation procedure: Ag electrode etched with chromic acid, then annealed in H2/Ar at 700 K with inductively heated furnace. # Short description of sample preparation. This can be very useful. :)

```